pkg/services/policer/policer.go

```diff
@@ -107,7 +107,7 @@ func defaultCfg() *cfg {
 		log:           zap.L(),
 		batchSize:     10,
 		rebalanceFreq: 1 * time.Second,
-		repCooldown:   0,
+		repCooldown:   1 * time.Second,
```

Comment:
this change should also be applied to the `cmd/config/policer` pkg

Reply:
Fixed.